src/features/list-view/__tests__/ListView-test.js

```diff
@@ -1,9 +1,28 @@
+// @flow
 import * as React from 'react';
 
 import ListView from '../ListView';
-
 // Global Declarations
-const rowData = [['', 'A', 'B', 'C'], ['', 'D', 'E', 'F']];
+
+const nameCellStyle = { height: 50, left: 0, position: 'absolute', top: 0, width: 300 };
```

Comment:
Could we reuse the style that's being passed in to the ListView component and import it here?

Reply:
Oof, that would be a little tricky.